vignettes/epipredict.Rmd

```diff
@@ -1,470 +1,565 @@
 ---
-title: "Get started with epipredict"
+title: "Get started with `epipredict`"
 output: rmarkdown::html_vignette
 vignette: >
-  %\VignetteIndexEntry{Get started with epipredict}
+  %\VignetteIndexEntry{Get started with `epipredict`}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
 
+# Introduction
+
 ```{r, include = FALSE}
-source("_common.R")
+source(here::here("vignettes/_common.R"))
 ```
 
-```{r setup, message=FALSE}
+```{r setup, message=FALSE, include = FALSE}
 library(dplyr)
 library(parsnip)
 library(workflows)
 library(recipes)
+library(epidatasets)
 library(epipredict)
+library(epiprocess)
+library(ggplot2)
+library(purrr)
+forecast_date <- as.Date("2021-08-01")
+used_locations <- c("ca", "ma", "ny", "tx")
```

Comment:
Consider making these visible in the code, so the vignette code can be reproduced by someone following along.

Reply:
Moved down a bit and done